common/app/templates/javaScriptConfig.scala.js

```diff
@@ -42,7 +42,7 @@
             "timingEvents": []
         },
         "libs": {
-            "foresee": "vendor/foresee/20150703/foresee-trigger.js",
+            "foresee": "/assets/javascripts/vendor/foresee/20150703/foresee-trigger.js",
```

Comment:
should this be `@Static("javascripts/vendor/foresee/20150703/foresee-trigger.js")`?